test/testReference.ts

```diff
@@ -29,12 +29,13 @@
 ///<reference path="components/plots/linePlotTests.ts" />
 ///<reference path="components/plots/areaPlotTests.ts" />
 ///<reference path="components/plots/barPlotTests.ts" />
+///<reference path="components/plots/clusteredBarPlotTests.ts" />
```

Comment:
Alphabetization, but now that I look at it again, a bunch of the others are also misalphabetized..